walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -24,9 +24,31 @@
 public final class FindMeetingQuery {
   /* Finds all time ranges that satisfy the request given the already existing set of events. */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    // Find potential conflicts between scheduled events and request.
-    List<TimeRange> potentialConflicts = getPotentialConflicts(events, request);
+    // Find potential conflicts between scheduled events and request using only
+    // mandatory attendees.
+    Collection<String> mandatoryAttendees = request.getAttendees();
+    List<TimeRange> mandatoryAttendeeConflicts = getPotentialConflicts(events, mandatoryAttendees);
```

Comment:
I guess you can use ImmutableList here?

Reply:
Ack - Here, I tried. I think I cannot because I am merging the two attendee lists and sorting the merged list in place.